setup.py

```diff
@@ -328,7 +328,7 @@ def get_proj_libraries():
             else:
                 extras_require[section].append(line.strip())
 install_requires = extras_require.pop('default')
-tests_require = extras_require.pop('tests', [])
+tests_require = extras_require.get('tests', [])
```

Comment:
I just wanted to pip install the test dependencies with ```pip install -e .[tests]``` rather than having to do something like ``python setup.py test`` (which I'm not 100% confident actually works).

Reply:
It's a deprecated entry point now, so we shouldn't really rely on it, too.